packages/overlay/src/vaadin-overlay-position-mixin.js

```diff
@@ -232,6 +232,11 @@ export const PositionMixin = (superClass) =>
 
       const targetRect = this.positionTarget.getBoundingClientRect();
 
+      if (targetRect.width === 0 && targetRect.height === 0 && this.opened) {
```

Comment:
Note: this condition can be potentially refactored to use `!this.positionTarget.checkVisibility() && this.opened`, but `checkVisibility` is only supported in Safari 17.4+.